Modules/Numerics/Statistics/test/itkWeightedCentroidKdTreeGeneratorTest9.cxx

```diff
@@ -39,7 +39,6 @@ itkWeightedCentroidKdTreeGeneratorTest9(int argc, char * argv[])
   using NumberGeneratorType = itk::Statistics::MersenneTwisterRandomVariateGenerator;
 
   auto randomNumberGenerator = NumberGeneratorType::New();
-  randomNumberGenerator->Initialize();
```

Comment:
For most types, `p->Initialize()` has no effect at all, when the object is just returned by `T::New()`. However, `MersenneTwisterRandomVariateGenerator::Initialize()` changes the seed, so it _does_ have an effect. So I'm not sure if it's OK to remove this particular `randomNumberGenerator->Initialize()` call. Or if it matters at all 🤔

Reply:
If New return object which will have a different seed each time, then this removal is fine. If New always starts with the same seed, then this removal of Initialize is a semantic change.

Reply:
I find it difficult to find out exactly what `MersenneTwisterRandomVariateGenerator::New()` does with its seed.  But it looks like the Initialize() call makes things even less predictable. I think I'll revert this removal, as my intention was just to removed "no op" Initialize() calls from the code.